internal/metrics/update.go

```diff
@@ -187,5 +203,115 @@ func updateMetrics(metrics map[string]*metricData) {
+		{"STATMQI/CONNDISC/MQDISC count", "mqdisc_count"},
+		{"STATMQI/OPENCLOSE/MQOPEN count", "mqopen_count"},
+		{"STATMQI/OPENCLOSE/Failed MQOPEN count", "failed_mqopen_count"},
+		{"STATMQI/OPENCLOSE/MQCLOSE count", "mqclose_count"},
+		{"STATMQI/OPENCLOSE/Failed MQCLOSE count", "failed_mqclose_count"},
+		{"STATMQI/INQSET/MQINQ count", "mqinq_count"},
+		{"STATMQI/INQSET/Failed MQINQ count", "failed_mqinq_count"},
+		{"STATMQI/INQSET/MQSET count", "mqset_count"},
+		{"STATMQI/INQSET/Failed MQSET count", "failed_mqset_count"},
+		{"STATMQI/PUT/Interval total MQPUT/MQPUT1 byte count", "interval_total_mqput_mqput1_bytes"},
+		{"STATMQI/PUT/Persistent message MQPUT count", "persistent_message_mqput_count"},
+		{"STATMQI/PUT/Failed MQPUT count", "failed_mqput_count"},
+		{"STATMQI/PUT/Non-persistent message MQPUT1 count", "non_persistent_message_mqput1_count"},
+		{"STATMQI/PUT/Persistent message MQPUT1 count", "persistent_message_mqput1_count"},
+		{"STATMQI/PUT/Failed MQPUT1 count", "failed_mqput1_count"},
+		{"STATMQI/PUT/Put non-persistent messages - byte count", "put_non_persistent_messages_bytes"},
+		{"STATMQI/PUT/Interval total MQPUT/MQPUT1 count", "interval_total_mqput_mqput1_count"},
+		{"STATMQI/PUT/Put persistent messages - byte count", "put_persistent_messages_bytes"},
+		{"STATMQI/PUT/MQSTAT count", "mqstat_count"},
+		{"STATMQI/PUT/Non-persistent message MQPUT count", "non_persistent_message_mqput_count"},
+		{"STATMQI/GET/Interval total destructive get- count", "interval_total_destructive_get_count"},
+		{"STATMQI/GET/MQCTL count", "mqctl_count"},
+		{"STATMQI/GET/Failed MQGET - count", "failed_mqget_count"},
+		{"STATMQI/GET/Got non-persistent messages - byte count", "got_non_persistent_messages_bytes"},
+		{"STATMQI/GET/Persistent message browse - count", "persistent_message_browse_count"},
+		{"STATMQI/GET/Expired message count", "expired_message_count"},
+		{"STATMQI/GET/Purged queue count", "purged_queue_count"},
+		{"STATMQI/GET/Interval total destructive get - byte count", "interval_total_destructive_get_bytes"},
+		{"STATMQI/GET/Non-persistent message destructive get - count", "non_persistent_message_destructive_get_count"},
+		{"STATMQI/GET/Got persistent messages - byte count", "got_persistent_messages_bytes"},
+		{"STATMQI/GET/Non-persistent message browse - count", "non_persistent_message_browse_count"},
+		{"STATMQI/GET/Failed browse count", "failed_browse_count"},
+		{"STATMQI/GET/Persistent message destructive get - count", "persistent_message_destructive_get_count"},
+		{"STATMQI/GET/Non-persistent message browse - byte count", "non_persistent_message_browse_bytes"},
+		{"STATMQI/GET/Persistent message browse - byte count", "persistent_message_browse_bytes"},
+		{"STATMQI/GET/MQCB count", "mqcb_count"},
+		{"STATMQI/GET/Failed MQCB count", "failed_mqcb_count"},
+		{"STATMQI/SYNCPOINT/Commit count", "commit_count"},
+		{"STATMQI/SYNCPOINT/Rollback count", "rollback_count"},
+	}
+
+	for _, mapping := range mappings {
```

Comment:
You don't have to do this. you can initialize golang maps with data.
 To initialize a map with some data, use a map literal:
```
commits := map[string]int{
    "rsc": 3711,
    "r":   2138,
    "gri": 1908,
    "adg": 912,
}
```